python/ray/serve/tests/test_logging.py

```diff
@@ -321,7 +557,8 @@ def __call__(self, *args) -> str:
 
     serve.run(A.bind())
 
-    r = httpx.get("http://localhost:8000/")
+    url = get_application_url(use_localhost=True)
+    r = httpx.get(f"{url}")
```

Comment:
```suggestion
    r = httpx.get(url)
```
let's also change this in all the other places too

Reply:
done.